Assets/Scripts/View/PauseMenu.cs

```diff
@@ -53,13 +54,14 @@ public void button1Click()
 	// Restart button was clicked
 	public void button2Click()
 	{
-
+		Engine.gamePaused = false;
+		SceneManager.LoadScene (SceneManager.GetActiveScene().name);
 	}
 
 	// Quit button was clicked
 	public void button3Click()
 	{
-
+		Application.Quit ();
```

Comment:
The quit button is not going to close the application. It will return to the menu once we have made the menu. That's why it wasn't an issue yet.